lib/paperclip/tempfile_factory.rb

```diff
@@ -13,7 +13,7 @@ def extension
     end
 
     def basename
-      Digest::MD5.hexdigest(File.basename(@name, extension))
+      Digest::SHA1.hexdigest(File.basename(@name, extension))
```

Comment:
Should this be the configuration option, or be rollbacked to `MD5` for backwards compatibility?